src/LazilyInitializedFields.jl

```diff
@@ -292,9 +288,8 @@ function lazy_struct(expr, mod)
     expr.args[1] = true # make mutable
     lazyfield = QuoteNode[]
     for (i, arg) in enumerate(body.args)
-        if arg isa Expr && arg.head === :macrocall && arg.args[1] === Symbol("@lazy")
-            body.args[i] = macroexpand(mod, arg)
-            name = body.args[i].args[1]
+        if arg isa Expr && arg.head === :macrocall && length(arg.args) == 3 && arg.args[1] === Symbol("@lazy")
+            name, body.args[i] = lazy_field(mod, arg.args[3])
```

Comment:
Looks like this is trying to call a function that doesn't exist (two arg `lazy_field`)?

Reply:
oops, yep, does not seem to need `mod` anymore
```suggestion
            name, body.args[i] = lazy_field(arg.args[3])
```